### DISCUSSION TITLE
账户突然无法使用

### DISCUSSION THREAD

**Discussion by guohui1992** [BurnHair/docs#106] [技术支持]
今天打开聊天框，发现无法聊天，无法签到，点击签到，会自动进入登录界面并且弹框提示
![image](https://github.com/BurnHair/docs/assets/22585034/b4481326-1e64-4f42-b3e3-f35bc220309a)
![image](https://github.com/BurnHair/docs/assets/22585034/ba96fa5c-192c-4260-a1b2-7bbfa8ca1972)

**Comment by BennyThink** [comment]
刷新网页，重新登录下。然后 令牌- 聊天 跳转过去的应该可以用的。网址是 https://nextchat.burn.hair 也可以单独窗口访问

**Comment by guohui1992** [comment]
1.点击令牌，依旧会跳转到登录界面
![image](https://github.com/BurnHair/docs/assets/22585034/1194c019-5943-4121-a068-c5222e0a58dd)
再次登录会提示错误
2.网页刷新好多遍了，edge和谷歌浏览器都是报一样的错。这账号不会封禁了吧

**Reply by BennyThink** [reply]
没有被封禁呀。你是不是改GitHub ID了？要不我帮你重置个密码，以后用密码登录就行

**Reply by BennyThink** [reply]
需要的话，发个邮件给我 我帮你重置下

**Comment by guohui1992** [comment]
好的，谢谢哈，账号就是guohui1992



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年6月14日(周五) 晚上10:48
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [BurnHair/docs] 账户突然无法使用 (Discussion #106)




 
需要的话，发个邮件给我 我帮你重置下

**Comment by guohui1992** [comment]
您邮箱是多少呢？首页那个超链谷歌浏览器点击无反应

**Reply by BennyThink** [reply]
i@burn.hair

**Reply by BennyThink** [reply]
已经发邮件给你了，记得改密码哦

**Comment by guohui1992** [comment]
现在可以了，感谢
